### PR TITLE
Fix particle image size

### PR DESCRIPTION
**Pull request by hpinkos**
All particle systems using the default `minimumImageSize` and `maximumImageSize` were using the same `Cartesian2` instance because we forgot to clone it.

### CODE REVIEW

File: Source/Scene/ParticleSystem.js [hunk 1]
```diff
@@ -134,8 +134,8 @@ define([
         this._minimumMass = defaultValue(options.mass, defaultValue(options.minimumMass, 1.0));
         this._maximumMass = defaultValue(options.mass, defaultValue(options.maximumMass, 1.0));
 
-        this._minimumImageSize = defaultValue(options.imageSize, defaultValue(options.minimumImageSize, defaultImageSize));
-        this._maximumImageSize = defaultValue(options.imageSize, defaultValue(options.maximumImageSize, defaultImageSize));
+        this._minimumImageSize = Cartesian2.clone(defaultValue(options.imageSize, defaultValue(options.minimumImageSize, defaultImageSize)));
+        this._maximumImageSize = Cartesian2.clonse(defaultValue(options.imageSize, defaultValue(options.maximumImageSize, defaultImageSize)));
```
Comment (mramato) [inline Source/Scene/ParticleSystem.js]: `clonse`?

Comment (hpinkos) [reply Source/Scene/ParticleSystem.js]: I got it right in my other repo 🤦‍♀️

**Comment by hpinkos** [comment]
@mramato ready

**Comment by hpinkos** [comment]
@ggetz @mramato bump

**Comment by hpinkos** [comment]
Can someone merge this?  @ggetz @mramato ?

**Comment by mramato** [comment]
Unit test?

**Comment by hpinkos** [comment]
@mramato ready

**Comment by mramato** [comment]
Thanks, I'll merge when green.

**Pull request merged**